pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test: TestTenantExecBuild_distsql_tenant_locality failed

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/3784f56c-a4ac-49a5-931b-f8831df67029?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [d2d974202781321ddb4fb1115c68fe7566ab9d06](https://github.com/cockroachdb/cockroach/commits/d2d974202781321ddb4fb1115c68fe7566ab9d06):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality2168262310
    test_log_scope.go:81: use -show-logs to present logs inline
[10:18:51] --- progress: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 2 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v0zAUfedXWOeJIVexO8GDn0BrEJG6bjSVQELVZOK7LsqHg-0wpir_HTlh0FVQdh8i3XvPPefkeA__rYZC-vl6-S5bsZeLLN_kH5dnLE-X6cWGvWLv11eXLLBPH9J1yioWQZKzc85en4GjtYZWuiEP9QUSHOfYcnTOFuS9dXG8H0GZ-QElOMq260McbzkK6whqj1CGmqCwsjPbJZHFUNBlPcIGDtuHP0c-6B1BzQ9UsgWUHPiBkDwttNFfa1qTNuQS8UQO4W246Sp6AMeFrfum9YpVnH0HR97p2CVSvElkIhPBdGvYnNlwR87jX07lkVPxxOl_IlmT72zr6VmZiCOlmRy2HGR2NL2Dt70r6NrZYsRO7dVINA4M-TBt51OTtY8rHxzp5rf9QyZ5kkmcYtpy3Nb2_qY0UBC_avaXz2MhHuidjxHld_Z-pN08dPEHb3XtieNSV7SgQK4p29KHsoAKrqdhePEzAAD__8r761E=
            
[10:18:53] --- done: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality2168262310
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (74.70s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34901

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/05b0e99b-d107-4e85-a3f1-e9a9d88b8559?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality94582701
    test_log_scope.go:81: use -show-logs to present logs inline
[09:44:20] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 2 statements
[09:44:22] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[09:44:22] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality94582701
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (80.74s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/fd2b1628-6e2d-4c90-a5c2-a9a4610deb39?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality3284105634
    test_log_scope.go:81: use -show-logs to present logs inline
[09:41:08] --- progress: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
[09:41:11] --- progress: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 5 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[09:41:12] --- done: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality3284105634
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (103.30s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/3c84960c-5927-477f-881f-5c6154b5b97d?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality2928678309
    test_log_scope.go:81: use -show-logs to present logs inline
[09:55:21] --- progress: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[09:55:21] --- done: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality2928678309
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (73.88s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/3248e418-de91-40d2-871c-3359f9d01e35?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [4f31fbae529e459c00ef2490e193c0c246e8d0f9](https://github.com/cockroachdb/cockroach/commits/4f31fbae529e459c00ef2490e193c0c246e8d0f9):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality3580781425
    test_log_scope.go:81: use -show-logs to present logs inline
[09:38:57] --- progress: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v0zAUfedXWOeJIVexO8GDn0BrEJG6bjSVQELVZOK7LsqHg-0wpir_HTlh0FVQdh8i3XvPPefkeA__rYZC-vl6-S5bsZeLLN_kH5dnLE-X6cWGvWLv11eXLLBPH9J1yioWQZKzc85en4GjtYZWuiEP9QUSHOfYcnTOFuS9dXG8H0GZ-QElOMq260McbzkK6whqj1CGmqCwsjPbJZHFUNBlPcIGDtuHP0c-6B1BzQ9UsgWUHPiBkDwttNFfa1qTNuQS8UQO4W246Sp6AMeFrfum9YpVnH0HR97p2CVSvElkIhPBdGvYnNlwR87jX07lkVPxxOl_IlmT72zr6VmZiCOlmRy2HGR2NL2Dt70r6NrZYsRO7dVINA4M-TBt51OTtY8rHxzp5rf9QyZ5kkmcYtpy3Nb2_qY0UBC_avaXz2MhHuidjxHld_Z-pN08dPEHb3XtieNSV7SgQK4p29KHsoAKrqdhePEzAAD__8r761E=
            
[09:38:57] --- done: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality3580781425
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (54.82s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/9a7a0fad-4521-4f50-8729-bf6d8f8789cb?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [57bd0a92676b1c58ee7e29979ce3dceb5837abdb](https://github.com/cockroachdb/cockroach/commits/57bd0a92676b1c58ee7e29979ce3dceb5837abdb):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality1852970600
    test_log_scope.go:81: use -show-logs to present logs inline
[10:17:08] --- progress: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[10:17:09] --- done: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality1852970600
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (64.88s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/f4a84354-673d-45b5-87e7-b12209e8eb4b?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality632184678
    test_log_scope.go:81: use -show-logs to present logs inline
[10:14:22] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[10:14:22] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality632184678
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (75.80s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/2c88008c-7b8d-494a-9a37-3f4860689843?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality899808695
    test_log_scope.go:81: use -show-logs to present logs inline
[09:49:11] --- progress: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v0zAUfedXWOeJIVexO8GDn0BrEJG6bjSVQELVZOK7LsqHg-0wpir_HTlh0FVQdh8i3XvPPefkeA__rYZC-vl6-S5bsZeLLN_kH5dnLE-X6cWGvWLv11eXLLBPH9J1yioWQZKzc85en4GjtYZWuiEP9QUSHOfYcnTOFuS9dXG8H0GZ-QElOMq260McbzkK6whqj1CGmqCwsjPbJZHFUNBlPcIGDtuHP0c-6B1BzQ9UsgWUHPiBkDwttNFfa1qTNuQS8UQO4W246Sp6AMeFrfum9YpVnH0HR97p2CVSvElkIhPBdGvYnNlwR87jX07lkVPxxOl_IlmT72zr6VmZiCOlmRy2HGR2NL2Dt70r6NrZYsRO7dVINA4M-TBt51OTtY8rHxzp5rf9QyZ5kkmcYtpy3Nb2_qY0UBC_avaXz2MhHuidjxHld_Z-pN08dPEHb3XtieNSV7SgQK4p29KHsoAKrqdhePEzAAD__8r761E=
            
[09:49:12] --- done: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality899808695
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (73.79s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/8d7319c1-b038-4e2a-87c6-2e99de81e84a?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality876566458
    test_log_scope.go:81: use -show-logs to present logs inline
[09:39:55] --- progress: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v0zAUfedXWOeJIVexO8GDn0BrEJG6bjSVQELVZOK7LsqHg-0wpir_HTlh0FVQdh8i3XvPPefkeA__rYZC-vl6-S5bsZeLLN_kH5dnLE-X6cWGvWLv11eXLLBPH9J1yioWQZKzc85en4GjtYZWuiEP9QUSHOfYcnTOFuS9dXG8H0GZ-QElOMq260McbzkK6whqj1CGmqCwsjPbJZHFUNBlPcIGDtuHP0c-6B1BzQ9UsgWUHPiBkDwttNFfa1qTNuQS8UQO4W246Sp6AMeFrfum9YpVnH0HR97p2CVSvElkIhPBdGvYnNlwR87jX07lkVPxxOl_IlmT72zr6VmZiCOlmRy2HGR2NL2Dt70r6NrZYsRO7dVINA4M-TBt51OTtY8rHxzp5rf9QyZ5kkmcYtpy3Nb2_qY0UBC_avaXz2MhHuidjxHld_Z-pN08dPEHb3XtieNSV7SgQK4p29KHsoAKrqdhePEzAAD__8r761E=
            
[09:39:55] --- done: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality876566458
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (74.99s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://tanzanite.cluster.engflow.com/invocations/default/6f2cff4d-c2d3-4903-8f72-e74af552ca31?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy8zbm9kZS10ZW5hbnQtbXVsdGlyZWdpb246M25vZGUtdGVuYW50LW11bHRpcmVnaW9uX3Rlc3Q=) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality3711618884
    test_log_scope.go:81: use -show-logs to present logs inline
[09:37:41] --- progress: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[09:37:41] --- done: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantExecBuild_distsql_tenant_locality3711618884
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (59.53s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13606831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13606831?buildTab=artifacts#/) on master @ [fcc065a5ea9161487f315c2dcafbd7cb20431597](https://github.com/cockroachdb/cockroach/commits/fcc065a5ea9161487f315c2dcafbd7cb20431597):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality1625469519
    test_log_scope.go:81: use -show-logs to present logs inline
[07:19:13] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[07:19:14] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality1625469519
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (54.76s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13618856?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13618856?buildTab=artifacts#/) on master @ [f740c076ec0d92972a47cf6633cf065c8c98678f](https://github.com/cockroachdb/cockroach/commits/f740c076ec0d92972a47cf6633cf065c8c98678f):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality2848105039
    test_log_scope.go:81: use -show-logs to present logs inline
    logic.go:2927: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3017/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[07:20:00] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3017/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3017/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3017/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality2848105039
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (43.62s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13629536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13629536?buildTab=artifacts#/) on master @ [3c837c2a86165188a26e8629b7e21b7ae0fb56ae](https://github.com/cockroachdb/cockroach/commits/3c837c2a86165188a26e8629b7e21b7ae0fb56ae):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality711354963
    test_log_scope.go:81: use -show-logs to present logs inline
[07:12:41] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4382/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality: 3 statements
    logic.go:2927: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4382/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v0zAUfedXWOeJIVexO8GDn0BrEJG6bjSVQELVZOK7LsqHg-0wpir_HTlh0FVQdh8i3XvPPefkeA__rYZC-vl6-S5bsZeLLN_kH5dnLE-X6cWGvWLv11eXLLBPH9J1yioWQZKzc85en4GjtYZWuiEP9QUSHOfYcnTOFuS9dXG8H0GZ-QElOMq260McbzkK6whqj1CGmqCwsjPbJZHFUNBlPcIGDtuHP0c-6B1BzQ9UsgWUHPiBkDwttNFfa1qTNuQS8UQO4W246Sp6AMeFrfum9YpVnH0HR97p2CVSvElkIhPBdGvYnNlwR87jX07lkVPxxOl_IlmT72zr6VmZiCOlmRy2HGR2NL2Dt70r6NrZYsRO7dVINA4M-TBt51OTtY8rHxzp5rf9QyZ5kkmcYtpy3Nb2_qY0UBC_avaXz2MhHuidjxHld_Z-pN08dPEHb3XtieNSV7SgQK4p29KHsoAKrqdhePEzAAD__8r761E=
            
[07:12:42] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4382/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4382/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4382/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality711354963
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (54.78s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test.TestTenantExecBuild_distsql_tenant_locality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13639612?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/13639612?buildTab=artifacts#/) on master @ [3c837c2a86165188a26e8629b7e21b7ae0fb56ae](https://github.com/cockroachdb/cockroach/commits/3c837c2a86165188a26e8629b7e21b7ae0fb56ae):


```
=== RUN   TestTenantExecBuild_distsql_tenant_locality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality3035423303
    test_log_scope.go:81: use -show-logs to present logs inline
    logic.go:2927: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4261/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:55: EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
        expected:
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJycklFv0zAQgN_5FdY9MeQqdsJ48BNoDSJS142mEkiomkx8K1HTONgOY6ry35FdYG3IUBs_RDrf5btPd96B_V6BgPTz7exdNicvp1m-zD_OLkieztKrJXlF3i9urokjnz6ki5RsiC_ilCSUXF4AhVornMstWhBfgAOFGCgksKLQGF2gtdr41C4UZuonCEahrJvW-esVhUIbBLEDV7oKQcBSfq1wgVKhiTxLoZNlFfDurbtrNvgIFK501W5rK8iGkh9AIW-kjyLO3kQ8uowYrDoKunVPjayTawTBD8yyKYi4o8_IPTm1tTYKDaojn1U3oD_XE91ESa9wWCXuqSRHKvz0ObGRc-JnzIkdycWny_GRcskZcvzZJQ7ILdA2urZ40o5Yr9OE-62jWuP-iVjdmgJvjS5C7T68CaBwodC6fZbvg6wOKe47GJTbv2_wkMT_S3p9RGKHJNYnxWOd_iElY0lhYveVfrgrFQhgv89k4PPngP9Brq1fW_5NPwTs8rHxQ7-XlUUK13KDU3RotmVdWlcWIJxpsete_AoAAP__yB-H8A==
            
        but found (query options: "") :
            distribution: full
            vectorized: true
            ·
            • scan
              missing stats
              table: t@t_pkey
              spans: [/1 - /1] [/3 - /3] [/5 - /5]
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJyMUV1v1DAQfOdXWPNEkU-xD8GDn0C9ICJdr-VyEkgoqky8vUb5cLAdSnXKf0dOKFwrONiHSLs7OzMZH-C_NlBIP12t32Yb9nyV5bv8w_qM5ek6Pd-xF-zd9vKCBfbxfbpNWc0iSHL2krNXZ-DorKGNbslDfYYExxIFR-9sSd5bF8eHCZSZ71CCo-r6IcRxwVFaR1AHhCo0BIWNXdg-iSyGgq6aCTZy2CH8PvJB7wlqeaSSraDkyI-E5Gmhnf7S0Ja0IZeIR3IIb8J1X9M9OM5tM7SdV6zm7Bs48l7HLpHidSITmQimO8OWzIZbch5_cyqfOBWPnP4jki353nae_isT8URpIceCg8ye5nfwdnAlXTlbTti5vZyIpoEhH-btcm6y7mHlgyPd_rJ_zCRPMolTTAXHTWPvrisDBfGzFn_4PBTigd77GFF-a-8m2t19H3_wRjeeOC50TSsK5Nqqq3yoSqjgBhrHZz8CAAD__8hg61A=
            
[07:21:02] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4261/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality with config 3node-tenant-multiregion: 7 tests, 1 failures
    logic.go:4005: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4261/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: error while processing
    logic.go:4005: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4261/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/3node-tenant-multiregion/3node-tenant-multiregion_test_/3node-tenant-multiregion_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality:68: too many errors encountered, skipping the rest of the input
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fc8838cec9d98939b6fc034fa2985021/logTestTenantExecBuild_distsql_tenant_locality3035423303
--- FAIL: TestTenantExecBuild_distsql_tenant_locality (45.30s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
 - <code>stress=true</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantExecBuild_distsql_tenant_locality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality
+++ b/pkg/sql/opt/exec/execbuilder/testdata/distsql_tenant_locality
@@ -52,7 +52,7 @@ SELECT id, locality FROM system.sql_instances WHERE locality IS NOT NULL ORDER B
 # - TableReader on SQL Instance 2 to scan Span /106/1/1/0
 # - TableReader on SQL Instance 3 to scan Span /106/1/3/0
 # - TableReader on SQL Instance 1 to scan Span /106/1/5/0.
-query T
+query T retry
 EXPLAIN (DISTSQL) SELECT * FROM t WHERE k IN (1, 3, 5)
 ----
 distribution: full

